src/zenml/integrations/gcp/flavors/vertex_experiment_tracker_flavor.py

```diff
@@ -0,0 +1,180 @@
+            Private services access must already be configured for the network.
+            If specified, all eligible jobs and resources created will be peered
+            with this VPC.
+        service_account:
+            Optional. The service account used to launch jobs and deploy models.
+            Jobs that use service_account: BatchPredictionJob, CustomJob,
+            PipelineJob, HyperparameterTuningJob, CustomTrainingJob,
+            CustomPythonPackageTrainingJob, CustomContainerTrainingJob,
+            ModelEvaluationJob.
+        encryption_spec_key_name:
+            Optional. The Cloud KMS resource identifier of the customer
+            managed encryption key used to protect a resource. Has the
+            form:
+            ``projects/my-project/locations/my-region/keyRings/my-kr/cryptoKeys/my-key``.
+            The key needs to be in the same region as where the compute
+            resource is created.
+        api_endpoint (str):
+                Optional. The desired API endpoint,
+                e.g., us-central1-aiplatform.googleapis.com
+        api_key (str):
+            Optional. The API key to use for service calls.
+            NOTE: Not all services support API keys.
+        api_transport (str):
+            Optional. The transport method which is either 'grpc' or 'rest'.
+            NOTE: "rest" transport functionality is currently in a
+            beta state (preview).
+        request_metadata:
+            Optional. Additional gRPC metadata to send with every client request.
+    """
+    location: Optional[str] = None
+    staging_bucket: Optional[str] = None
+    network: Optional[str] = None
+    service_account: Optional[str] = None
+    encryption_spec_key_name: Optional[str] = None
+    api_endpoint: Optional[str] = None
+    api_key: Optional[str] = None
+    api_transport: Optional[str] = None
+    request_metadata: Optional[dict] = None
+
+
+class VertexExperimentTrackerFlavor(BaseExperimentTrackerFlavor):
+    """Flavor for the VertexAI experiment tracker."""
+
+    @property
+    def name(self) -> str:
+        """Name of the flavor.
+
+        Returns:
+            The name of the flavor.
+        """
+        return GCP_VERTEX_EXPERIMENT_TRACKER_FLAVOR
+
+    @property
+    def docs_url(self) -> Optional[str]:
+        """A URL to point at docs explaining this flavor.
+
+        Returns:
+            A flavor docs url.
+        """
+        return self.generate_default_docs_url()
+
+    @property
+    def sdk_docs_url(self) -> Optional[str]:
+        """A URL to point at SDK docs explaining this flavor.
+
+        Returns:
+            A flavor SDK docs url.
+        """
+        return self.generate_default_sdk_docs_url()
+
+    @property
+    def logo_url(self) -> str:
+        """A URL to represent the flavor in the dashboard.
+
+        Returns:
+            The flavor logo.
+        """
+        # TODO: replace with a proper logo
```

Comment:
@nkhusainov Do you know of any special logo for the experiment tracking part of Vertex, or is it just the default Vertex one?

Reply:
Looks like there is no special logo for Experiment Tracker

Reply:
I added the same logo to `https://public-flavor-logos.s3.eu-central-1.amazonaws.com/experiment_tracker/vertexai.png`, that way we can replace it in the future in case they decide to have separate logos for these services